src/pe/utils.rs

```diff
@@ -147,6 +147,59 @@ pub fn try_name<'a>(
     }
 }
 
+/// Safe version of try_name that handles packed binaries gracefully
+pub fn safe_try_name<'a>(
```

Comment:
is this a function you use? I don't see it used anywhere, and i'd prefer not to add more freestanding `pub` functions unless absolutely necessary.